main.cpp

```diff
@@ -227,31 +222,26 @@ struct IntType
     IntType& powInternal(int myNumber);
 };
 
-IntType& IntType::add(int myNumber)
+IntType& IntType::operator+=(int myNumber)
 {
     *value += myNumber;
     return *this;
 }
 
-IntType& IntType::subtract(int myNumber)
+IntType& IntType::operator-=(int myNumber)
 {
     *value -= myNumber;
     return *this;
 }
 
-IntType& IntType::multiply(int myNumber)
+IntType& IntType::operator*=(int myNumber)
 {
-    *value *= myNumber;
+    *value /= myNumber;
     return *this;
 }
 
-IntType& IntType::divide(int myNumber)
+IntType& IntType::operator/=(int myNumber)
 {
-    if (myNumber == 0)
-    {
-        std::cout << "You're dividing by 0!" << std::endl;
-        return *this; 
-    }
```

Comment:
Just because you change to operator/= doesn't mean the functionality should change.   your divide/assign function body should stay the same.

- [ ] revert to the body with the warning for your `operator/=`